share/completions/git.fish

```diff
@@ -1426,10 +1426,22 @@ complete -f -c git -n '__fish_git_using_command reset; and not contains -- -- (c
 
 ### restore
 complete -f -c git -n '__fish_git_needs_command' -a restore -d 'Restore working tree files'
-complete -f -c git -n '__fish_git_using_command restore' -s s -l source -d 'Specify the source tree used to restore working tree'
+complete -f -c git -n '__fish_git_using_command restore' -s s -l 'source=' -d 'Specify the source tree used to restore the working tree'
```

Comment:
I'm pretty sure this works both as `--source=SOURCE` and `--source SOURCE`. (If not, this would be a weird inconsistency)